static/js/filter.js

```diff
@@ -231,7 +232,7 @@ Filter.parse = function (str) {
     var operand;
     var term;
 
-    var matches = str.match(/"[^"]+"|\S+/g);
+    var matches = str.match(/(\S+:)?"[^"]+"?|\S+/g);
```

Comment:
Ok, I think I like that. We haven't supported having a space there in the past, but there's no situation where you'd want to have an empty operand anyway.